theme.json

```diff
@@ -8,6 +8,9 @@
   "assets": [
     "scss/index.scss"
   ],
+  "references": [
+    "widgetInstances.$.uuid:widgetInstance"
```

Comment:
Why not call the reference type `:widgetUuid`? It saves any confusion in guessing what `widgetInsance` refers to.

Reply:
@tonytlwu because we refer to the entity, not the actual property target for some properties like `mediaFile`. However, if you believe it still creates confusion I'm fine with changing this to `widgetInstanceUuid`

Reply:
What do you mean by the entity? Is `widgetInstances.$.uuid` referring to an object? It's odd that an entity is called a UUID.

Reply:
Feel free to share a sample of the data structure.

Reply:
@tonytlwu see https://github.com/Fliplet/fliplet-widget-list-large-thumbs/blob/master/widget.json#L44

This refers to a `mediaFile` entity and the target is the `id`, although the whole object gets updated.

What I am saying is that `widgetInstance` refers to the widget entity, and `uuid` is the target property on which the object is matched, although other properties of the object are updated (as for the `mediaFile`)

Reply:
OK, it's like I thought then. The reason we wouldn't do this for the other reference types, e.g. page, media, dataSource etc., is that those have IDs and that's it. If I was to follow that same logic, I'd think `widgetInstance` refers to the widget instance ID. But that's not true here. You've made an exception so it's best to make it specific without needing to learn it either by constantly referring to a document or making repeated mistakes.

Reply:
@tonytlwu I've updated both PRs to use the more specific `widgetInstanceUuid`